.github/workflows/lint.yaml

```diff
@@ -27,6 +27,13 @@ jobs:
       with:
         go-version-file: go.mod
 
+    # https://github.com/golangci/golangci-lint-action?tab=readme-ov-file#verify
```

Comment:
Wdyt about adding `verify: true` in current golangci lint action instead of creating a new one ?

Reply:
Changed. First impression was, that it does not support both operation, but looks like it does. Validated here https://github.com/golangci/golangci-lint-action/blob/3b4f037d0e94e85d98f9824ef87b2dc32d53fbd5/src/run.ts#L140